### ISSUE TITLE
Microphone still broken in Mac dmg release 1.85.6

### ISSUE THREAD

**Issue by ejolly** [psychopy/psychopy#1654]
[This issue](https://discourse.psychopy.org/t/microphone-doesnt-work/2934/3) which was resolved in this https://github.com/psychopy/psychopy/commit/68df98967d491a70f2c002ba372bdf1419e1fc86 somehow doesn't seem to have made it into the the most recent [Mac OSX dmg release](https://github.com/psychopy/psychopy/releases).

**Comment by ejolly** [comment]
I can confirm that it works on standalone version 1.82.0

**Comment by peircej** [comment]
Thanks. I'm not quite sure what happened here. The code is correctly fixed in the master branch and in the 1.85 release branch. It must have been an error in the packaging but I don't know why. As this is a demo though you can simply change the code and I'll look at it again for the next release.

**Comment by ejolly** [comment]
Just to provide feedback on the latest 1.90 dev release. There seems to be a new issue on macOS when even trying to load the microphone. Here's the error output just running the demo. Testing on the same machine that microphone works fine using 1.82.0:
```
##### Running: /Applications/PsychoPy2.app/Contents/Resources/lib/python2.7/psychopy/demos/coder/input/latencyFromTone.py #####
pyo version 0.8.6 (uses single precision)
Traceback (most recent call last):
  File "/Applications/PsychoPy2.app/Contents/Resources/lib/python2.7/psychopy/demos/coder/input/latencyFromTone.py", line 14, in <module>
    from psychopy import microphone, sound, core, visual, event
  File "/Applications/PsychoPy2.app/Contents/Resources/lib/python2.7/psychopy/microphone.py", line 43, in <module>
    from psychopy import core, logging, sound, web, prefs
  File "/Applications/PsychoPy2.app/Contents/Resources/lib/python2.7/psychopy/sound/__init__.py", line 74, in <module>
    from . import backend_pyo as backend
  File "/Applications/PsychoPy2.app/Contents/Resources/lib/python2.7/psychopy/sound/backend_pyo.py", line 18, in <module>
    import sounddevice
  File "sounddevice.pyc", line 354, in <module>
  File "/Applications/PsychoPy2.app/Contents/Resources/lib/python2.7/cffi/api.py", line 139, in dlopen
    lib, function_cache = _make_ffi_library(self, name, flags)
  File "/Applications/PsychoPy2.app/Contents/Resources/lib/python2.7/cffi/api.py", line 769, in _make_ffi_library
    backendlib = _load_backend_lib(backend, libname, flags)
  File "/Applications/PsychoPy2.app/Contents/Resources/lib/python2.7/cffi/api.py", line 758, in _load_backend_lib
    return backend.load_library(name, flags)
OSError: cannot load library /Applications/PsychoPy2.app/Contents/Resources/lib/python2.7/site-packages.zip/_sounddevice_data/libportaudio.dylib: dlopen(/Applications/PsychoPy2.app/Contents/Resources/lib/python2.7/site-packages.zip/_sounddevice_data/libportaudio.dylib, 2): no suitable image found.  Did find:
    /Applications/PsychoPy2.app/Contents/Resources/lib/python2.7/site-packages.zip/_sounddevice_data/libportaudio.dylib: stat() failed with errno=20
```

**Comment by mgroth0** [comment]
just reported related (same?) issue [on the forum](https://discourse.psychopy.org/t/error-cannot-load-library-when-running-a-basic-stroop-task/3874)

Copied post from forum:

> 
mgroth05m
Hi, I’m also totally new to PsychoPy. I know some coding but nothing about loading dylibs. Tried running from both the coder and the builder and got the same exact error message, word for word.

OS Mac OSX 10.13.2
PsychoPy version : 1.90.dev1
Standard Standalone? (y/n) : Yes

@Cochonnet any progress on this? Found this help page for the dlopen function, but I didn’t find anything in there referring to the “no suitable image found”. I noticed the path to the dylib contains a zip file, is this normal? I also looked inside site-packages.zip and found no folder called “_sounddevice_data”, so my best guess is that this is the problem.
